guides/common/attributes-foreman-el.adoc

```diff
@@ -1,2 +1,3 @@
 // Overrides for foreman-el build
-:dnf-modules: foreman:el8
+:dnf-module: foreman:el8
```

Comment:
I thought it'd be possible to use `{project-context}` here, but it's missing for katello: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-katello.adoc

I am unsure if this is an issue to begin with.